Docs/pages/usage/verifying.md

```diff
@@ -0,0 +1,92 @@
+# Verifying Stone Proof Using the Binary
+
+This guide will walk you through how to verify a Stone proof using the provided binaries. It includes steps for both setting up the environment using Docker and manually installing the required dependencies.
+
+## Creating and Verifying a Proof of a Cairo Program
+
+1. Navigate to the Example Test Directory:
+   • Start by navigating to the Cairo example test directory.
+
+   ```bash
+   cd e2e_test/Cairo
+   ```
+
+2. Install cairo-vm/cairo1-run:
+   • Clone and install the necessary dependencies for cairo-vm:
+   ```bash
+   git clone https://github.com/lambdaclass/cairo-vm.git
+   cd cairo-vm/cairo1-run
+   make deps
+   ```
+
+# Docker-based Setup
```

Comment:
You don't need a Dockerfile, you already have docker images, check this https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#creating-and-verifying-a-test-proof-using-binaries

Reply:
> You don't need a Dockerfile, you already have docker images, check this https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#creating-and-verifying-a-test-proof-using-binaries

Again, please, read https://github.com/dipdup-io/stone-packaging/blob/master/README.md

Reply:
Okay